CMakeLists.txt

```diff
@@ -135,13 +132,14 @@ SET(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
 SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 SET(CMAKE_INSTALL_RPATH "$ORIGIN:$ORIGIN/lib:$ORIGIN/lib/plugins:$ORIGIN/../lib:$ORIGIN/../lib/plugins")
 
+SET(BLA_STATIC ON)
+SET(OPENSSL_USE_STATIC_LIBS TRUE)
```

Comment:
Can we keep this openssl static variable as a CMake flag instead of hard-coded? I think this would make things harder on developers that just have apt-get install curl4-openssl-dev

Reply:
I changed to be set only if it is not defined already, which makes it static by default but allows one to override that setting. However, I have to say that you might run into issues if you build against one version but then have it pick up another version dynamically. I haven't had a chance to look very closely, but you can give it a try if you feel like it.